checker/check.ml

```diff
@@ -242,10 +242,6 @@ let try_locate_absolute_library dir =
 
 let try_locate_qualified_library lib = match lib with
 | PhysicalFile f ->
-  let () =
-    if not (System.file_exists_respecting_case "" f) then
-      error_lib_not_found { dirpath = []; basename = f; }
-  in
```

Comment:
IIUC after this patch a missing file would fail with an uncaught exception in intern_from_file. Can't we get a more user-readable error still?

Reply:
If you mean on non existing files we get
~~~
[intern /tmp/fo.vo ...
 failed!]

Fatal Error: User error: /tmp/fo.vo: /tmp/fo.vo: No such file or directory. Try to rebuild it.
~~~

Reply:
That's a bit subpar, couldn't we fix the check instead?

Reply:
Ping @SkySkimmer

Reply:
changed